dockers/docker-orchagent/supervisord.conf

```diff
@@ -59,3 +59,9 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:arp_update]
+command=/usr/bin/arp_update
+autostart=true
```

Comment:
>true [](start = 10, length = 4)

suggest follow Joe's pattern:
1. /usr/bin/start.sh is the only section autostart
2. /usr/bin/start.sh will start all other sections, so the order is guaranteed